account_payment_term_extension/tests/test_account_payment_term_multi_day.py

```diff
@@ -188,114 +193,114 @@ def test_out_invoice_amount_untaxed_payment_term(self):
         self.assertEqual(invoice.line_ids[3].debit, 600.0)
 
     def test_invoice_normal_payment_term(self):
-        invoice = self._create_invoice(self.payment_term_0_day_5, "2020-01-01", 10, 100)
+        invoice = self._create_invoice(self.payment_term_0_day_5, "2023-01-01", 10, 100)
         invoice.action_post()
         for line in invoice.line_ids:
             if line.date_maturity:
                 self.assertEqual(
-                    fields.Date.to_string(line.date_maturity),
-                    "2020-02-05",
+                    line.date_maturity.strftime("%Y-%m-%d"),
```

Comment:
are the `fields.Date.to_string` -> `strftime("%Y-%m-%d")` changes really needed?

Reply:
rollbacked to v15 version